README.md

```diff
@@ -127,24 +127,37 @@ Consider using [rlink] instead of `yarn link`
 
 ### Tests
 
-Tests are run by [mocha] under the hood, and written using [chai] and [sinon]. You can easily run the tests suite with:
```

Comment:
You should remove `mocha`, `chai` and `sinon` link anchors at the end of the document unless they are used elsewhere in this doc